### PR TITLE
UserStory1001

### PR DESCRIPTION
**Pull request by AndrewChetvertakov**

### CODE REVIEW

**Review by ihsan-onal — APPROVED** [review]
pew pew

File: src/test/java/AndrewWorking/UserStory1001.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        int start = 0;
+        int end = pew.length-1;
+        char temp;
+
+        while(start<end){
+            temp = pew[start];
+            pew[start] = pew[end];
+            pew[end] = temp;
+            start++;
```
Comment (maximvoronov757) [inline src/test/java/AndrewWorking/UserStory1001.java:16]: waiting to see how to resolve conflicts

**Review by maximvoronov757 — COMMENTED** [review]
> pew pew

kwa-kwa-kwa?

**Review by MDogan87 — APPROVED** [review]
looks good

**Pull request merged**